hardware_interface/doc/fake_components_userdoc.rst

```diff
@@ -0,0 +1,36 @@
+.. _fake_components_userdoc:
+
+Fake Components
+----------------
+Fake components are trivial "simulations" of the hardware components, i.e., System, Sensor, and Actuator.
+They provide ideal behavior by mirroring commands to their states.
+The implement corresponding hardware interface can be laded instead of real hardware for offline testing of ros2_control framework.
```

Comment:
Should we remove `implement` .?

```suggestion
The corresponding hardware interface can be laded instead of real hardware for offline testing of ros2_control framework.
```

Reply:
Could also fix "laded" ;)